docker/Dockerfile

```diff
@@ -58,39 +68,20 @@ RUN if [ "$(uname -m)" != "aarch64" ]; then \
     && \
     rm -rf /var/lib/apt/lists/* /tmp/*
 
-# TODO: just keep the backtrace, colorbt, rust_log, and cargo_home variables as those are the only needed at build time.
-# Build arguments and variables set to change how tests are run, tracelog levels,
-# and Network to be used (Mainnet or Testnet)
+# Build arguments and variables set for tracelog levels and debug information
 #
 # We set defaults to all variables.
 ARG RUST_BACKTRACE
 ENV RUST_BACKTRACE=${RUST_BACKTRACE:-0}
 
-ARG RUST_LIB_BACKTRACE
-ENV RUST_LIB_BACKTRACE=${RUST_LIB_BACKTRACE:-0}
-
 ARG COLORBT_SHOW_HIDDEN
 ENV COLORBT_SHOW_HIDDEN=${COLORBT_SHOW_HIDDEN:-0}
```

Comment:
```suggestion
ENV COLORBT_SHOW_HIDDEN=${COLORBT_SHOW_HIDDEN:-1}
```

Reply:
Fixed here: 
[fix(docker): enable backtraces for errors and panics](https://github.com/ZcashFoundation/zebra/pull/7200/commits/37b591441125d53824746ea4a2abc4fa60cdb51b)